icu4c/source/python/icutools/databuilder/__main__.py

```diff
@@ -83,12 +83,6 @@
     default = False,
     action = "store_true"
 )
-flag_parser.add_argument(
-    "--ignore_xml_deprecates",
```

Comment:
Should we keep obsolote options for one or more releases as no-ops, with the help text giving the status?
It does not seem necessary to break users who specify the option.
Or better to make them fail fast so they know this is no longer an option?

Reply:
I just added the option in #642.  It isn't in ICU 64.2.

Reply:
I added the option in #642 to give users (Google) a way to disable the feature.  But now that I'm proposing to make this the default behavior, I'm deleting the option I added.  If we wanted to give a way for users to get the old behavior, we could make an option named `--use-xml-deprecates`.

Reply:
ok no concern then. tnx.